### PR TITLE
Project2- News site

### PR DESCRIPTION
**Pull request by Pauan8**

### CODE REVIEW

File: code/about.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <meta charset="ut-8" />
+    <meta name="viewport" content="width=device-width">
+    <title>About Us</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<header class="simple-header">
+    <nav class="nav">
+        <input type="checkbox" id="checkbox">
+        <label for="checkbox"><a class="menu-button"></a>
```
Comment (EmOlsz) [inline code/about.html:14]: It's not the best idea to wrap `a` tag inside `label`, moreover it's not best idea to add `input` and `label` inside navigation. Let's delete them!

File: code/about.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        </label>
+        <div class="menu-content">
+            <a href="index.html" target="blank">home</a>
```
Comment (EmOlsz) [inline code/about.html:17]: Great idea to implement subpages! Nonetheless, proper value for `target` attribute is `_blank`, not `blank` ;)

File: code/about.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        </div>
+    </nav>
+    <div class="logo"><img src="./pictures/logo_space.svg"></div>
+    <div class="logo-simple"><img src="./pictures/logo_space_simple.svg"></div>
+</header>
+
+<footer class="footer">
+    <div class="footer-wrapper">
+        <p>&copy; Pauline Andersson 2021</p>
+        <container class="footer-img-container">
```
Comment (EmOlsz) [inline code/about.html:30]: `<container />` tag doesn't exists, we shouldn't use it, this needs to be fixed.
Instead, consider using `<div />` tag!

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,205 @@
...
+        <a href="about.html" target="blank">about</a>
+        <a href="signup.html" target="blank">sign up</a>
+        <a href="login.html" target="blank">login</a>
+      </div>
+      <div class="search-content">
+        <input type="checkbox" id="searchbox">
+        <div class="searchbar">
+          <input type="text" placeholder="Search..">
+        </div>
+        <label for="searchbox">
```
Comment (EmOlsz) [inline code/index.html:30]: We shouldn't use `label` tags as wrappers for images (or any different tag than `input`). They should only me implemented in two possible setups :

1. Label connected with input via id:
```
<input type="text" placeholder="Search..." id="search-input" />
<label for="search-input">Type phrase in searchbox</label>
```

1. Label connected with input via nesting:
```
<label>
    Type phrase in searchbox
    <input type="text" placeholder="Search..." />
</label>
```

Let's fix that!

File: code/index.html [hunk 5]
```diff
@@ -1,22 +1,205 @@
...
+    <!-- meteor animation -->
+    <div class="meteor"></div>
+    <!-- logo, normal and simpler version for mobile devices-->
+    <div class="logo"><img src="./pictures/logo_space.svg"></div>
+    <div class="logo-simple"><img src="./pictures/logo_space_simple.svg"></div>
+  </header>
+
+  <!--Top news, the most important news. Take up big space and top of the content-->
+  <section class="big-news">
+    <container class="big-img-container">
```
Comment (EmOlsz) [inline code/index.html:43]: Once again, let's change all `<container />` occurances to valid HTML tags (for example `div`'s) ;)

File: code/index.html [hunk 6]
```diff
@@ -1,22 +1,205 @@
...
+      <img src="./pictures/astronaut.jpg">
+    </container>
+    <container class="big-text-container">
+      <h2>TOP NEWS</h2>
+      <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Id adipisci eum ipsa, facere atque ea a aut eligendi,
+        rerum placeat, quibusdam recusandae saepe reiciendis! Magni minus repudiandae eligendi sequi minima!
+        <br>
```
Comment (EmOlsz) [inline code/index.html:50]: We should avoid using `<br />` tags, you can read more about them in feedback document! (link attached in the main comment) - needs to be fixed

File: code/index.html [hunk 7]
```diff
@@ -1,22 +1,205 @@
...
+    </div>
+    <div class="grid-wrapper">
+      <container class="small-img-container">
+        <img src="./pictures/sun.jpg">
+      </container>
+    </div>
+    <div class="grid-wrapper">
+      <h3>Small news</h3>
+      <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Id adipisci eum ipsa, facere atque ea a aut
+        eligendi, rerum placeat, quibusdam recusandae saepe reiciendis! <br><a href="i" target="blank">More
```
Comment (EmOlsz) [inline code/index.html:156]: Let's fix all of the wrong indentations as well (details in feedback document)

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,710 @@
...
+  align-items: center;
+  justify-content: space-evenly;
+  border-bottom: solid #f1df77 1px;
+  background-color: rgba(0, 0, 0, 0.425);
+  position: fixed;
+  z-index: 1;
+}
+
+/* necessary for doing drop-down menu that appears on click in mobile version */
+#checkbox,
```
Comment (EmOlsz) [inline code/style.css:117]: We shouldn't style by id's, needs to be fixed as well! (details in feedback document)

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Hey Pauline!

First of all, I'm impressed by meteor animation!!!

Your project is well done, all resolutions are supported, great job :) Nonetheless, there are some things that needs to be fixed (take a look on single comments). After providing changes, I will be able to approve project.

We have prepared for all of you a document with complete set of feedback regarding ideal project for this week, check it out and try to take those tips with you to next projects!
https://www.notion.so/technigo/General-feedback-for-class-Jan-2021-ac7821ff3e1849c3bcac26f5d9888230

**Comment by Pauan8** [comment]
Hi! I just have some questions:


+<head>
+    <meta charset="ut-8" />
+    <meta name="viewport" content="width=device-width">
+    <title>About Us</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<header class="simple-header">
+    <nav class="nav">
+        <input type="checkbox" id="checkbox">
+        <label for="checkbox"><a class="menu-button"></a>

It's not the best idea to wrap a tag inside label, moreover it's not best
idea to add input and label inside navigation. Let's delete them!

If I delete them the "checkbox-hack" to make a drop down menu och click
with only CSS won't work?


> +    <nav class="nav">
+      <input type="checkbox" id="checkbox">
+      <label for="checkbox"><a class="menu-button"></a>
+      </label>
+      <div class="menu-content">
+        <a href="contacts.html" target="blank">contact</a>
+        <a href="about.html" target="blank">about</a>
+        <a href="signup.html" target="blank">sign up</a>
+        <a href="login.html" target="blank">login</a>
+      </div>
+      <div class="search-content">
+        <input type="checkbox" id="searchbox">
+        <div class="searchbar">
+          <input type="text" placeholder="Search..">
+        </div>
+        <label for="searchbox">

We shouldn't use label tags as wrappers for images (or any different tag
than input). They should only me implemented in two possible setups :

   1. Label connected with input via id:

<input type="text" placeholder="Search..." id="search-input" />
<label for="search-input">Type phrase in searchbox</label>


   1. Label connected with input via nesting:

<label>
    Type phrase in searchbox
    <input type="text" placeholder="Search..." />
</label>

Let's fix that!

Same here. This solution is only used to get usability onCLick on mobile
with only CSS. If I take it away I remove the function.

But I can take it away, but it was intentionally done to get this function.


 +    </div>
+    <div class="grid-wrapper">
+      <h3>Astronaut William Thornton dead</h3>
+      <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.
jewhjkklhwk jhekjhwkhk hgwkhkwhklhglw<br>
+        <a href="h" target="blank">More -&gt;</a>
+      </p>
+    </div>
+    <div class="grid-wrapper">
+      <container class="small-img-container">
+        
+      </container>
+    </div>
+    <div class="grid-wrapper">
+      <h3>Small news</h3>
+      <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Id
adipisci eum ipsa, facere atque ea a aut
+        eligendi, rerum placeat, quibusdam recusandae saepe
reiciendis! <br><a href="i" target="blank">More

Let's fix all of the wrong indentations as well (details in feedback
document)

I use an extension called "beautify" for intendations. I should not use
this extension?

Den tors 28 jan. 2021 kl 11:46 skrev Maksymilian Olszewski <
notifications@github.com>:

File: code/about.html (hunk 1 above)
Comment (Pauan8) [reply code/about.html:14]: If I delete them the "checkbox-hack" to make a drop down menu och click
with only CSS won't work? Should I remove this function then? I know it is a bit of a "hack", but I liked making the page working well on mobile with only css

**Comment by Pauan8** [comment]
Requested changes done.

File: code/about.html (hunk 1 above)
Comment (EmOlsz) [reply code/about.html]: Ahh, true, I didn't quite catch that one. In that case let's leave it as it is ;)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**